apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestDataPathsActivity.java

```diff
@@ -143,6 +143,8 @@ public class TestDataPathsActivity  extends BaseAutoGlitchActivity {
     private double mPhaseErrorSum;
     private double mPhaseErrorCount;
 
+    private boolean skipRemainingTests;
```

Comment:
I think of it as an imperative command, not a "should".
I can't think of another data type that fits the name. It could only be a boolean.
I will add the m prefix.